flujo/telemetry/otel_hook.py

```
@@ -33,6 +36,164 @@
+
+                loop = asyncio.get_running_loop()
+            except RuntimeError:
+                loop = None
+            try:
+                if loop is not None:
+                    loop.create_task(self._backend.save_spans(run_id, spans))
+                    return
+                result = self._backend.save_spans(run_id, spans)
+            except NotImplementedError:
+                return
+            except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)